test/python/pulse/test_cmd_def.py

```diff
@@ -83,9 +83,14 @@ def test_parameterized_schedule(self):
         cmd_def.add('pv_test', 0, converted_instruction)
         self.assertEqual(cmd_def.get_parameters('pv_test', 0), ('P1', 'P2'))
 
-        sched = cmd_def.get('pv_test', 0, P1='0', P2=-1)
```

Comment:
I believe the error is because the argument is supplied as a string. I am surprised this was passing in the first place.  I think it should pass if this is switched to a float input.

Reply:
Yes, I thought it was supposed to work, and it works in current version of python, but not 3.5 for some reason. I thought it was due to symbols being not really recognised, but it seems not. I tried to do a float input before, and Travis IC found no issue, but the other IC test failed, even though they did the same thing, so I am a bit confused. I'm installing a python3.5 build to do the test on my side.